lib/font.js

```diff
@@ -0,0 +1,31 @@
+const camelCase = require('camelcase');
+
+// based on https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Common_weight_name_mapping
+const FONT_WEIGHT = {
+  thin: 100,
+  extraLight: 200,
+  light: 300,
+  book: 350, // exceptional, book variants should not be used for devices, but just in case
+  normal: 400,
+  medium: 500,
+  semiBold: 600,
+  bold: 700,
+  extraBold: 800,
+  black: 900,
+};
+
+const fontWeights = Object.keys(FONT_WEIGHT).map(weightId => ({
```

Comment:
Why not just create an array, to begin with? Then the mapping is not needed and the order is guaranteed.

Reply:
see getFontWeight(), I think using FONT_WEIGHT.normal is clearer than fontWeights[3]